webapp/src/components/Staking/Modal/StakingApprovalModalInfo.tsx

```diff
@@ -98,20 +57,21 @@ const StakingApprovalModalInfo: React.FC<StakingApprovalModalInfoProps> = ({
   onApprove,
 }) => {
   const logo = useMemo(() => {
-    switch (getAssets(vaultOption)) {
-      case "WBTC":
-        return <GreenWBTCLogo />;
-      case "USDC":
-        return <GreenUSDCLogo />;
+    const asset = getAssets(vaultOption);
+    const Logo = getAssetLogo(asset);
+
+    switch (asset) {
+      case "WETH":
+        return <Logo height="48px" />;
       default:
-        return <GreenWETHLogo height="48px" />;
+        return <Logo />;
```

Comment:
Why do we have a special case for WETH?

Reply:
The reason is WETH asset do not come with margin. Otherwise it looks ugly